frontend/public/components/sidebars/explore-type-sidebar.tsx

```diff
@@ -39,6 +39,8 @@ export const ExploreType: React.FC<ExploreTypeProps> = (props) => {
     ? currentSelection.path
     : [getDefinitionKey(kindObj, allDefinitions)];
   const currentDefinition: SwaggerDefinition = _.get(allDefinitions, currentPath) || {};
+  const currentProperties =
+    _.get(currentDefinition, 'properties') || _.get(currentDefinition, 'items.properties');
```

Comment:
I would suggest optional chaining here, except it's easier to backport if we use `_.get`. So I'm good with this.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining